packages/editor-ui/src/App.vue

```diff
@@ -262,9 +239,7 @@ export default defineComponent({
 		await this.redirectIfNecessary();
 		void this.checkForNewVersions();
 		await this.checkForCloudPlanData();
-		await this.initBanners();
-
-		void this.checkForCloudPlanData();
+		void this.initBanners();
```

Comment:
as discussed in grooming, would be great to add tests that the upgrade bar shows up

Reply:
yes! Will do. Wanted to make sure the approach was correct before adding the tests!

Reply:
thanks